### ISSUE TITLE
Alerting: Panel read-only mode, supports display of Alert list

### ISSUE THREAD

**Issue by GitHamburg** [grafana/grafana#79869]
**Why is this needed**:

At present, panels that have already been associated with alerts are only in editing mode and support displaying alert lists.

![grafana-panel-edit-alert](https://github.com/grafana/grafana/assets/16780183/88e4fa2a-f506-4a1d-931b-8f7556ef3c60)

In daily life, when it comes to permissions or viewing mode, entering panel view mode cannot see the Alert list.

**What would you like to be added**:

The alert panel is in view mode and also supports viewing the alert list.

**Who is this feature for?**

Dashboard and Alerting users.

**Comment by ppcano** [comment]
I believe this is a feature request rather than a bug.

@GitHamburg, if I understood your issue correctly, you want to access the panel alerts while the panel is in view mode. Is that correct?

For context, a panel is often in view mode in the following common scenarios:

- A user with a **Viewer** role accesses the dashboard.
- The dashboard is in **Read-only** mode.


In both cases, the behavior is the same:

- Alert thresholds and statuses are displayed within the panel, as documented [here](https://grafana.com/docs/grafana/latest/alerting/alerting-rules/link-alert-rules-to-panels/).
- The Alert list is not accessible from the dashboard, as users cannot edit the panel.

![Image](https://github.com/user-attachments/assets/d5c2751b-8fec-41a9-a523-47b42c115965)


## Workaround


Currently, the solution is to navigate to the Alert rules view and filter by the specific dashboard.


![Image](https://github.com/user-attachments/assets/b9f9f318-b2a6-48b7-b62e-add8b89f9a0a)


## Suggested solution

A possible enhancement could be to add a "View alert rules" option in the More… menu, which redirects to the Alert rule list page with the relevant filters applied. However, filtering by `panelID` is not currently supported.


![Image](https://github.com/user-attachments/assets/8c83ca6a-31a3-4975-9a3f-3e5587f6422c)

**Comment by GitHamburg** [comment]
@ppcano Yes, it's a feature. You got it, thanks for your suggested solution !
